lib/rust/types/src/dim_macros.rs

```diff
@@ -1,8 +1,6 @@
 //! Macros allowing generation of swizzling getters and setters.
 //! See the docs of [`build.rs`] and usage places to learn more.
 
-
-
```

Comment:
Can be excluded, I suppose

Reply:
https://github.com/enso-org/enso/pull/3939 :(